webpack/webpack.config.js

```diff
@@ -0,0 +1,38 @@
+var ExtractTextPlugin = require('extract-text-webpack-plugin');
```

Comment:
I'd like us to use ES6 also for the configuration, and webpack is smart enough to load/parse the file with babel if you add `babel` to the file name ( `webpack.config.babel.js`).  What do you think, should we give it a try?